Sources/MessagePackEncoder.swift

```diff
@@ -9,11 +9,14 @@
 import Foundation
 
 open class MessagePackEncoder: Encoder {
+    public var allowIntKeys: Bool
     public var codingPath: [CodingKey] = []
     public var userInfo: [CodingUserInfoKey : Any] = [:]
     fileprivate var storage = MessagePackStorage()
 
-    public init() {}
+    public init(allowIntKeys: Bool = false) {
```

Comment:
Thanks for a good idea!
But I want to change to:

1. Add KeyEncodingType.
```
public enum KeyEncodingType {
    case string
    case int
}
```

2. Change the initializer.
```
    private let keyEncodingType: KeyEncodingType

    public init(keyEncodingType: KeyEncodingType = .string) {
        self.keyEncodingType = keyEncodingType
    }
```

Reply:
Sounds like a good idea.